Emby.Server.Implementations/Data/SqliteUserDataRepository.cs

```diff
@@ -375,5 +375,15 @@ private UserItemData ReadRow(IReadOnlyList<IResultSetValue> reader)
 
             return userData;
         }
+
+        /// <inheritdoc/>
+        /// <remarks>
+        /// There is nothing to dispose here since <see cref="BaseSqliteRepository.WriteLock"/> and
+        /// <see cref="BaseSqliteRepository.WriteConnection"/> are managed by <see cref="SqliteItemRepository"/>.
+        /// See <see cref="Initialize(IUserManager, SemaphoreSlim, SQLiteDatabaseConnection)"/>.
+        /// </remarks>
+        protected override void Dispose(bool dispose)
+        {
+        }
```

Comment:
You can see where the write lock and write connection properties are assigned to in this class [here](https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/Data/SqliteUserDataRepository.cs#L37). The `Initialize()` method is called by the `SqliteItemRepository` [here](https://github.com/jellyfin/jellyfin/blob/master/Emby.Server.Implementations/Data/SqliteItemRepository.cs#L328), which passes its own internal write lock and connection to be shared by this repo